### PR TITLE
Get friends list

### PR DESCRIPTION
**Pull request by Ahmed-anwar**

### CODE REVIEW

File: src/main/java/com/linkedin/replica/connections/database/handlers/impl/ArangoFriendsListHandler.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    /**
+     * Returns user's friends list
+     *
+     * @param userID
+     */
+    @Override
+    public UserInFriendsList[] getFriendsList(String userID) {
+        String collectionName = Configuration.getInstance().getArangoConfigProp("collection.users.name");
+        String query = "For u IN " + collectionName + " FILTER u.userId == @id LET result = ( FOR u2 in "+ collectionName + " FILTER u2.userId in u.friendsList return {userId: u2.userId, firstName : u2.firstName, lastName : u2.lastName, profilePictureUrl : u2.profilePictureUrl} ) return result";
```
Comment (mostafa-abdullah) [inline src/main/java/com/linkedin/replica/connections/database/handlers/impl/ArangoFriendsListHandler.java]: Split to multiple lines

File: src/main/java/com/linkedin/replica/connections/database/handlers/FriendsHandler.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package com.linkedin.replica.connections.database.handlers;
+
+import java.sql.SQLException;
+
+public abstract class FriendsHandler implements DatabaseHandler {
```
Comment (mostafa-abdullah) [inline src/main/java/com/linkedin/replica/connections/database/handlers/FriendsHandler.java]: Interface

File: src/main/java/com/linkedin/replica/connections/database/handlers/FriendsListHandler.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package com.linkedin.replica.connections.database.handlers;
+
+import com.linkedin.replica.connections.models.UserInFriendsList;
+
+import java.util.ArrayList;
+
+public abstract class FriendsListHandler implements DatabaseHandler{
```
Comment (mostafa-abdullah) [inline src/main/java/com/linkedin/replica/connections/database/handlers/FriendsListHandler.java]: interface

**Review by mostafa-abdullah — CHANGES_REQUESTED** [review]

**Comment by Ahmed-anwar** [comment]
@mostafa-abdullah  resolved.

**Review by mostafa-abdullah — APPROVED** [review]

**Pull request merged**